pages/osx/security.md

```diff
@@ -13,4 +13,12 @@
 
 - Create a keychain:
 
-`security create-keychain -p {{password}} {{name.keychain}}`
+`security create-keychain -p {{password}} {{keychain.name}}`
+
+- Set a certificate to use with a website or [s]ervice by its [c]ommon name (this will fail if several certificates with the same common name exist):
+
+`security set-identity-preference -s {{URL|hostname|service}} -c {{friendly_certificate_name_in_keychain}} {{path/to/file.keychain}}`
```

Comment:
I'd put the common name in quotes, because it often contains spaces.

```suggestion
`security set-identity-preference -s {{URL|hostname|service}} -c "{{common_name}}" {{path/to/file.keychain}}`
```